lib/std/start.zig

```diff
@@ -437,7 +437,7 @@ fn expandStackSize(phdrs: []elf.Phdr) void {
 }
 
 fn callMainWithArgs(argc: usize, argv: [*][*:0]u8, envp: [][*:0]u8) u8 {
-    std.os.argv = argv[0..argc];
+    if (builtin.os.tag != .windows or builtin.link_libc) std.os.argv = argv[0..argc];
```

Comment:
This will never be called when `builtin.os.tag == .windows and !builtin.link_libc`